packages/alea-frontend/components/ExamSchedule.tsx

```diff
@@ -0,0 +1,64 @@
+import { Box, Typography } from '@mui/material';
+import QuizIcon from '@mui/icons-material/Quiz';
+
+export default function ExamSchedule({ examDates }) {
+  if (!examDates?.length) return null;
+
+  const content = examDates.map((exam, idx) => {
+    const hasTime = exam.examStartTime && exam.examEndTime;
+
+    const start = new Date(`${exam.examDate}T${exam.examStartTime || '00:00'}:00+02:00`);

```

Comment:
add a comment explain that +0200 is used because we assume Europe/Berlin time.

btw, I will ask rakesh to manage timezones properly in the next update. then this assumption will not be needed anymore